### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by oscarhjern**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,18 +44,26 @@ We plan on using the following technologies to complete this project:
...
 - Attach one ultrasonic sensor to the right side of the car, connect the trig pin to D33 & the echo pin to D18
-- >LIDAR SENSOR
+- Attach one MICRO LIDAR SENSOR directly to the left side of the car
 - Attach the GPS module to the car and connect the GPS RXPin to D16 and the TXPin to D17
 
 #### Calibration
-- > ADD CALIBRATION
+To calibrate the car, please see the documentation provided by Platisd (https://github.com/platisd/smartcar_shield)
+
+Sensors that need calibiration:
```
Comment (WhiteMaken) [inline README.md]: Fix typo in "calibration".

File: README.md [hunk 2]
```diff
@@ -44,18 +44,26 @@ We plan on using the following technologies to complete this project:
...
-- > ADD CALIBRATION
+To calibrate the car, please see the documentation provided by Platisd (https://github.com/platisd/smartcar_shield)
+
+Sensors that need calibiration: 
+- Gyroscope 
+- Odometer
+- Ultrasonic sensors
+- LIDAR SENSOR
+
+Methods that need calibiration:
```
Comment (WhiteMaken) [inline README.md]: Fix typo in "calibration".

**Review by WhiteMaken — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -44,18 +44,26 @@ We plan on using the following technologies to complete this project:
 #### Sensors
 - Attach one ultrasonic sensor to the front of the car, connect the trig pin to D19 & the echo pin to D5
 - Attach one ultrasonic sensor to the right side of the car, connect the trig pin to D33 & the echo pin to D18
-- >LIDAR SENSOR
+- Attach one MICRO LIDAR SENSOR directly to the left side of the car
```
Comment (WhiteMaken) [inline README.md]: maybe micro lidar sensor should not be uppercase? looks a bit out of place.

File: README.md [hunk 4]
```diff
@@ -44,18 +44,26 @@ We plan on using the following technologies to complete this project:
...
 
 #### Calibration
-- > ADD CALIBRATION
+To calibrate the car, please see the documentation provided by Platisd (https://github.com/platisd/smartcar_shield)
+
+Sensors that need calibiration: 
+- Gyroscope 
+- Odometer
+- Ultrasonic sensors
+- MICRO LIDAR SENSOR
```
Comment (WhiteMaken) [inline README.md]: maybe micro lidar sensor should not be uppercase? looks a bit out of place.

File: README.md [hunk 5]
```diff
@@ -84,10 +94,10 @@ We plan on using the following technologies to complete this project:
...
-## Developer
-- Oscar Hjern
-- Mikael Höije
-- Jakob Sixten
-- Johann Tammen
-- Maximilien Uddgren
-- Maja Wennroth
+## Developers
+- [Oscar Hjern](https://github.com/oscarhjern), SCRUM Master 
+- [Maja Wennroth](https://github.com/majawennroth), Car Wizard
```
Comment (majawennroth) [inline README.md]: I would prefer car responsibel

**Review by majawennroth — APPROVED** [review]

File: README.md [hunk 6]
```diff
@@ -84,10 +94,13 @@ We plan on using the following technologies to complete this project:
...
-- Johann Tammen
-- Maximilien Uddgren
-- Maja Wennroth
+## Developers
+- [Oscar Hjern](https://github.com/oscarhjern), SCRUM Master 
+- [Maja Wennroth](https://github.com/majawennroth), Car Responsible   
+- [Mikael Höije](https://github.com/mikaelhoije), App Guy 
+- [Jakob Sixten](https://github.com/sijakob), Connection Specialist 
+- [Johann Tammen](https://github.com/johann-tam), Git Master 
+- [Maximilien Uddgren](https://github.com/WhiteMaken), App Guy
```
Comment (WhiteMaken) [inline README.md:106]: Everyone has cool titles but Mike and me have "app guys", so sad :(

**Review by WhiteMaken — APPROVED** [review]

**Review by majawennroth — APPROVED** [review]

**Pull request merged**